tests/integration/api/enable_params/test_enable_6.php

```diff
@@ -45,7 +45,15 @@
 ]
 */
 
-newrelic_enable_params(array());
+/*
+ * This throws a TypeError on PHP 8. To simplify, lets just catch the error and continue
+ * execution to allow this test to run on any PHP version
```

Comment:
Does the test description still hold when run under PHP 8?
> Calling newrelic_enable_params() with an argument that is neither a boolean nor an integer enables the recording of request parameters.

Reply:
You're right. I've skipped the test on PHP 8+ since the error halts execution. Catching and continuing doesn't really do anything in PHP 8+ so it makes more sense to skip. I've added a more detailed description as well.